app/common/utils/chains.ts

```diff
@@ -0,0 +1,6 @@
+export const DEFAULT_CHAINS: Record<string, ChainId> = {
```

Comment:
do you think it's utils, not constants?

Reply:
Seems valid, I'll move it as you suggest